server/lib/brain/index.js

```diff
@@ -8,7 +9,10 @@ const { getReply } = require('./brain.getReply');
 const { load } = require('./brain.load');
 
 const Brain = function Brain() {
-  this.nlpManager = new NlpManager({ languages: SUPPORTED_LANGUAGES });
+  this.nlpManager = new NlpManager({
+    forceNER: true,
```

Comment:
what is this option 'forceNER' doing? Just asking :) Couldn't find the doc

Reply:
I don't, i have see this on the npm 'node-nlp' page 
![Screenshot 2021-02-15 at 21 33 24](https://user-images.githubusercontent.com/11317212/107990805-84b36a80-6fd5-11eb-8a07-12b271649a1a.png)
But I found nothing about that into doc and code. I have removed that :)